auth/cmd/main/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/p-society/gCSB/auth/pkg/constants"
+)
+
+func main() {
+
+	r := mux.NewRouter()
+	srv := &http.Server{
+		Handler:      r,
+		Addr:         "127.0.0.1:8000",
+		WriteTimeout: 15 * time.Second,
+		ReadTimeout:  15 * time.Second,
+	}
+	r.HandleFunc("/ping", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("pong"))
+	}).Methods("GET")
+	http.Handle("/", r)
+	log.Fatal(srv.ListenAndServe())
+}
```

Comment:
The Auth Module won't be a different service. It will just have 2 functions as of now that verifies and creates tokens as discussed